server/src/main/java/io/druid/server/initialization/ZkPathsConfig.java

```diff
@@ -80,6 +80,11 @@ public String getCoordinatorPath()
     return (null == coordinatorPath) ? defaultPath("coordinator") : coordinatorPath;
   }
 
+  public String getOverlordPath()
+  {
+    return defaultPath("overlord");
```

Comment:
this is different than the other patterns which have a settable path, why does this one need to require only the default path?

Reply:
I understand the reasoning, but this should be consistent with the other items in the class. If it is not needed to have settable paths, then this is not the right PR to make such a change.

Reply:
earlier we supported `druid.zk.paths.indexer.leaderLatchPath` property which was being used by overlord and is obtained via `IndexerZkConfig`. (I'm gonna note this in the release notes)
even if I honor that property in this PR, still overlord is incompatible given that underlying algorithm for leader election changes. Given that compatibility is broken anyway, I don't see the value in making it configurable just because all the other properties in this class are such due to historical reasons. In face, I would say any newly added property added to this class shouldn't be made configurable.

do you still feel strongly about making if configurable ? :)